tests/mask.py

```diff
@@ -99,10 +99,10 @@ def setUp(self):
         except Exception:
             self.maskFile = None
         else:
-            if True:
-                self.maskFile = os.path.join(dataDir, "small_MI.fits")
-            else:
-                self.maskFile = os.path.join(dataDir, "871034p_1_MI.fits")
+            self.maskFile = os.path.join(dataDir, "small_MI.fits")
+            # Above mask looks like this, possibly left-shifted by some number of mask planes.
```

Comment:
I don't understand this comment (what does "looks like this" mean?)

More generally I'm not thrilled with silently changing behavior based on whether afwdata is setup. In my opinion a test should either need it or not. If a test needs afwdata then it should be explicitly skipped if afwdata is missing, so it's obvious to users that it was skipped.